tox.ini

```diff
@@ -47,3 +47,5 @@ deps =
 commands =
   black --check docs_guide/ example_docs/ qiskit_sphinx_theme/ tests/ setup.py
   isort --check docs_guide/ example_docs/ qiskit_sphinx_theme/ tests/ setup.py
+  mypy --install-types
```

Comment:
This is bad. We don't want to dynamically install dependencies. They should all be pinned in requirements-dev.txt.

But this shouldn't be necessary once you update the ignore list in pyproject.toml correctly.